src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -27,13 +28,13 @@ public abstract class UbiquitousLanguageAsFile implements UbiquitousLanguageRepo
   public void saveAll(Set<UbiquitousLanguage> ubiquitousLanguages) {
     try {
       Files.createDirectories(destinationFolder);
+      Manifest manifest = new Manifest();
 
       for (UbiquitousLanguage ubiquitousLanguage : ubiquitousLanguages) {
-        UbiquitousLanguageDto dto = UbiquitousLanguageDto.from(ubiquitousLanguage);
-        String fileName = dto.getName().toLowerCase().replace(" ", "_");
-        serializer.writeValue(destinationFolder.resolve(fileName + "." + extension).toFile(), dto);
+        manifest.addFile(writeToFile(ubiquitousLanguage).toString());
       }
```

Comment:
What do you think about something like that:

```java
Manifest.generateFrom(listOfPaths)
```

where listOfPaths are the ubiquitous language file paths previously generated.

This way you'll have something like that:

```java
List<Path> filePaths = ubiquitousLanguages.stream()
  .map(UbiquitousLanguageAsFile::saveToFile)
  .collect(toList());

Manifest.generateFrom(filePaths);
```

No more `addFile`, no more `ArrayList`